zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/UsageMetricHandler.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import static io.camunda.webapps.schema.entities.metrics.UsageMetricsEventType.EDI;
+import static io.camunda.webapps.schema.entities.metrics.UsageMetricsEventType.RPI;
+
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.metrics.UsageMetricsEntity;
+import io.camunda.webapps.schema.entities.metrics.UsageMetricsEventType;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.UsageMetricIntent;
+import io.camunda.zeebe.protocol.record.value.UsageMetricRecordValue;
+import io.camunda.zeebe.protocol.record.value.UsageMetricRecordValue.EventType;
+import io.camunda.zeebe.util.DateUtil;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public record UsageMetricHandler(String indexName)
+    implements ExportHandler<UsageMetricsEntity, UsageMetricRecordValue> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(UsageMetricHandler.class);
+  private static final String ID_PATTERN = "%s_%s";
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.USAGE_METRIC;
+  }
+
+  @Override
+  public Class<UsageMetricsEntity> getEntityType() {
+    return UsageMetricsEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(final Record<UsageMetricRecordValue> record) {
+    return record.getIntent() != null
```

Comment:
I think this can be simplified since UsageMetricIntent.EXPORTED.equals(intent) will not be true on null values and if not instance of UsageMetricIntent

```suggestion
    return UsageMetricIntent.EXPORTED.equals(intent)
&& !record.getValue().getEventType().equals(EventType.NONE);
```